user_keymaps/kdb424/nyquist_r2.py

```diff
@@ -33,8 +39,8 @@
 TAB_SB = KC.LT(5, KC.TAB)
 SUPER_L = KC.LM(4, KC.LGUI)
 
-# ---------------------- Keymap ---------------------------------------------------------
```

Comment:
Not totally opposed to trashing the comment, but left for visual reason. Mostly curious why it's gone.

Reply:
Reverting. I deleted it as I had a moment, but I decided to leave it in for the other files as _it's not my config_.
In general I hope we can move towards closer alignment in boards/ and user_keymaps as exemplars for the community.

Reply:
I'm honestly almost in favor of moving user keymaps out to it's own repo so when things break, go out of date, ect, it's less code in the primary repo, but that's a totally different topic.

Reply:
Agree, on both points. I'm trying to maintain focus in this PR as well.